pages.bn/android/settings.md

```diff
@@ -0,0 +1,20 @@
+# settings
+
+> Android OS সম্পর্কে তথ্য পান।
+> আরও তথ্য পাবেন: <https://adbinstaller.com/commands/adb-shell-settings-5b670d5ee7958178a2955536>.
+
+- `global` নামস্থানে সেটিংসের একটি তালিকা প্রদর্শন করুন:
+
+`settings list {{গ্লোবাল}}`
+
+- একটি নির্দিষ্ট সেটিং এর মান পান:
+
+`settings get {{global}} {{airplane_mode_on}}`
```

Comment:
```suggestion
- একটি নির্দিষ্ট সেটিং এর একটি মান পান:

`settings get {{global}} {{airplane_mode_on}}`
```

Optional fix (error source: original page): unexpected `the` term, expected `a`.
Note: `Get a value of a specific setting:` can be used as a translation source.

Reply:
There is only one value of a setting at a time. So what's wrong with `the` here?

Reply:
Value can be anything (valid, of course),  that's why I suggested to use `a`. I use `the` just when there is just value possible. You can ignore this suggestion as it's optional. 😄